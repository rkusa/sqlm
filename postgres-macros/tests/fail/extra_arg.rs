fn main() {
    sqlm_macros::sql!("{}", 1, 2);
}
