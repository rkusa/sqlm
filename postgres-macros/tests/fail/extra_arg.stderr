error: argument never used
 --> tests/fail/extra_arg.rs:2:32
  |
2 |     sqlm_macros::sql!("{}", 1, 2);
  |                                ^
